server/sonar-web/src/main/js/apps/issues/components/AppContainer.js

```diff
@@ -50,7 +50,7 @@ const fetchIssueOrganizations = issues => dispatch => {
   );
 };
 
-const fetchIssues = (query: RawQuery) => dispatch =>
+const fetchIssues = (query /*: RawQuery */) => dispatch =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)